src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Build/LanguageServiceErrorListProvider.cs

```diff
@@ -25,7 +25,7 @@ internal partial class LanguageServiceErrorListProvider : IVsErrorListProvider
         private IVsLanguageServiceBuildErrorReporter2 _languageServiceBuildErrorReporter;
 
         [ImportingConstructor]
-        public LanguageServiceErrorListProvider(UnconfiguredProject project, ILanguageServiceHost host)
+        public LanguageServiceErrorListProvider(ILanguageServiceHost host)
```

Comment:
Does removing `UnconfiguredProject` imports here and elsewhere change the scope of the export? Even though they're unused references, perhaps they have an impact on the lifecycle of the components.

https://github.com/Microsoft/VSProjectSystem/blob/master/doc/overview/scopes.md

Reply:
Good question.  You've spooked me enough that I am going to back these out as I have no idea how to rationalize implicit scopes here.  @davkean should we add comments for apparently unnecessary imports like this?